config/lava/chromeos/chromeos-tast-template.jinja2

```diff
@@ -0,0 +1,45 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% include "/chromeos/chromeos-extra-args.jinja2" %}
```

Comment:
Does this need to be in a separate file? Can it just be defined in this file directly?